servers_v7.json

```
@@ -108,4 +108,8 @@
     "name": "Eradicationdustry",
     "address": ["n1.yeet.ml:6577"]
   }
+  {
+    "name": "[red]LamNguyen[green]Server
```

```suggestion
    "name": "[red]LamNguyen[green]Server"
```

Hey mate, you have a missing `"`.